packages/terra-application-links/src/tabs/_ApplicationTabMenuContent.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import List from 'terra-list';
+import styles from './ApplicationTabMenuContent.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The child tabs to be placed in the menu.
+   */
+  children: PropTypes.node,
+  /**
+   * Prop from popup, determines if the menu height is bound by the screen.
+   */
+  isHeightBounded: PropTypes.bool,
+  /**
+   * Prop from popup, determines if the menu width is bound by the screen.
+   */
+  isWidthBounded: PropTypes.bool,
+  /**
+   * Ref callback used by the popup for do positioning.
+   */
+  refCallback: PropTypes.func,
+};
+
+const ApplicationTabMenuContent = ({
+  children,
+  isHeightBounded,
+  isWidthBounded,
+  refCallback,
+  ...customProps
+}) => {
+  const listStyle = {};
+  if (isHeightBounded) {
```

Comment:
Should we use isHeightBounded & isWidthBounded to append css classes instead of passing listStyle?

Reply:
Good call!